x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/transport/SecurityTransportExceptionHandler.java

```diff
@@ -30,12 +30,7 @@ public void accept(TcpChannel channel, Exception e) {
             // just close and ignore - we are already stopped and just need to make sure we release all resources
             CloseableChannel.closeChannel(channel);
         } else if (SSLExceptionHelper.isNotSslRecordException(e)) {
-            if (logger.isTraceEnabled()) {
```

Comment:
Is it okay to change this? The @elastic/es-security team when they added this seemed to prefer the trace then fallback to warn style. It seems odd that this PR modifies this just for this specific log message?

Reply:
I don't see any reason why we want to keep the stack even under trace logging.
I think it was included defensively when we started trying to silence some of these common place exceptions, but iI don't know of any case where we would have needed it.